rapids_build_backend/impls.py

```diff
@@ -118,37 +118,25 @@ def _get_git_commit() -> Union[str, None]:
 
 
 @contextmanager
-def _edit_git_commit(config):
+def _write_git_commit(config, project_name: str):
```

Comment:
Can we document (briefly) how `project_name` is used and why it's relevant for writing a git commit?